core/src/main/java/org/stellar/anchor/sep24/Sep24TransactionBuilder.java

```diff
@@ -44,17 +44,17 @@ public Sep24TransactionBuilder completedAt(long time) {
   }
 
   public Sep24TransactionBuilder stellarAccount(String stellarAccount) {
-    txn.setStellarAccount(stellarAccount);
+    txn.setSep10Account(stellarAccount);
     return this;
   }
 
   public Sep24TransactionBuilder stellarAccountMemo(String accountMemo) {
-    txn.setAccountMemo(accountMemo);
+    txn.setSep10AccountMemo(accountMemo);
     return this;
   }
```

Comment:
👀  I think you forgot to update the name of these builder methods.

Reply:
Good catch. 👍 

Fixed.